### PR TITLE
Rework conspiracy objectives

### PR DESCRIPTION
**Pull request by TobleroneSwordfish**
[REWORK] [INPUT WANTED] [REMOVAL]
## About the PR 
Removes a lot of the weirdly specific and non-open ended objectives and adds a few new ones.
Adds disposals and crew lounge as meetup points.

Adds:
- Start a cult worshipping ["Heisenbee", "Morty", "Dr. Acula", "Monsieur Stirstir", "Jones the cat", "Sylvester", "Hooty McJudgementowl", "the AI", "Discount Dan", "the engine"]. ("the engine" is disabled on Oshan)
- Convince the crew that the station is underwater. (or in space if the station actually is underwater)
- Replace as many heads of staff with imposters as possible.
- Transfer ownership of the station to Discount Dan. Ensure all the crew are loyal and the station is branded correctly.
- Set yourselves up as vigilantes and arrest people for made up crimes.
- Pose as a team of undercover Nanotrasen inspectors and make an example out of anyone you deem incompetent or too competent at their job.
- Turn as much of the station as you can into [material1] and [material2], including anyone who gets in your way. (material1 and 2 chosen from all ore types, plus glass, water, rubber, rock, flesh and coral if on an underwater map)
- Remind the crew of their own mortality by stockpiling as many of their organs as you can.
- Replace the employees of an entire department with conspirators.
- Make sure all available food and water is chemically spiked.
- Turn command against each other for petty reasons.

Please suggest more!
## Why's this needed? 
Conspirator is my favourite antag, but the objectives are very hit or miss.
I've removed a lot here, so feel free to tell me to re-add any you like, or suggest new ones.

## Changelog

```changelog
(u)LeahTheTech
(*)Reworked conspiracy objectives.
```

### CODE REVIEW

**Comment by TobleroneSwordfish** [comment]
Marking as ready for review since it's been around for two weeks and no-one seems to have anything more to add. Let me know if I should remove the old objectives completely or leave them commented out.

File: code/datums/objective.dm [hunk 1]
```diff
@@ -1184,24 +1192,87 @@ ABSTRACT_TYPE(/datum/objective/conspiracy)
...
+					continue
+				materials += material.comname
+		else
+			materials += list("Glass", "Water", "Rubber", "Rock", "Flesh")
+#ifdef UNDERWATER_MAP
+		//but if we're underwater there's always a decent chance it's coral
+		materials += list("Coral", "Coral", "Coral")
+#endif
+		for (var/material in materials)
+			boutput(world, "[material]")
```
Comment (Xkeeper0) [inline code/datums/objective.dm:1257]: i smell accidental debug output!!

File: code/datums/objective.dm [hunk 2]
```diff
@@ -1184,24 +1192,87 @@ ABSTRACT_TYPE(/datum/objective/conspiracy)
...
+// 	explanation_text = "Explain to the crew how, yes, Heisenbee really was framed by the Chompski brothers."
 
 /datum/objective/conspiracy/centcom
 	explanation_text = "Convince the crew that Central Command has forsaken them."
 
 /datum/objective/conspiracy/spacelaw
 	explanation_text = "Establish and enforce a set of station protocols and policies."
 
+/datum/objective/conspiracy/discountdan
+	explanation_text = "Transfer ownership of the station to Discount Dan. Ensure all the crew are loyal and the station is branded correctly."
```
Comment (Xkeeper0) [inline code/datums/objective.dm:1214]: one suggestion: this, but the syndicate. possibly other factions? get some wizards in here?

**Comment by Xkeeper0** [comment]
i looked over this;  the worst objectives are gone and the new ones provide ample opportunity for active subterfuge and/or murder. big improvements

File: code/datums/objective.dm (hunk 1 above)
Comment (TobleroneSwordfish) [reply code/datums/objective.dm:1257]: ah shit

File: code/datums/objective.dm (hunk 2 above)
Comment (TobleroneSwordfish) [reply code/datums/objective.dm:1214]: There's already "Convince the crew to forsake Nanotrasen and join the Syndicate.", but I've added "the Space Wizards Federation" and "Bombini"

**Pull request merged**